### PR TITLE
Games service communication: Information-service

### PR DESCRIPTION
**Pull request by CaoimheCahill**
Fixing naming issue that wasn't working correctly when communicating between information service and game service.
All methods tested and work.
Unit tests added for InformationServiceClient.

### CODE REVIEW

**Review by CaoimhinHawk1 — APPROVED** [review]
Good work Caoimhe, good job on sorting the issue

File: game-service/src/test/resources/.env
Comment (Whomstt) [inline game-service/src/test/resources/.env:1]: env file should not be committed. is this an additional env file to the one at the root of game service?

**Review by Whomstt — APPROVED** [review]
Looks good to me, just one comment about the env

File: game-service/src/test/resources/.env
Comment (CaoimheCahill) [reply game-service/src/test/resources/.env:1]: This is just an .env file used for testing purposes only. Doesn't have any sensitive information

Comment (Whomstt) [reply game-service/src/test/resources/.env:1]: ah okay all good then nevermind

**Pull request merged**